jena-arq/src/main/java/org/apache/jena/sparql/util/QueryExecUtils.java

```diff
@@ -191,6 +197,11 @@ private static void doSelectQuery(Prologue prologue, QueryExecution qe, ResultsF
         outputResultSet(results, prologue, outputFormat) ;
     }
 
+    private static void doJsonQuery(Prologue prologue, QueryExecution queryExecution, ResultsFormat outputFormat) {
+        JsonArray results = queryExecution.execJson();;
```

Comment:
Double `;`.  Nit-pick, as it's normally harmelss (unless after some "end" statements like `return`, `break`, etc, I think?)